tests/test_pytypes.cpp

```diff
@@ -9,6 +9,13 @@
 
 #include "pybind11_tests.h"
 
+#include <array>
+#include <deque>
+#include <list>
+#include <set>
+#include <string>
+#include <vector>
+
```

Comment:
```suggestion
```

I assume these are leftovers?

Reply:
Yes, but leave the `#include <string>` as strings are used.